src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeed.cs

```diff
@@ -236,14 +253,18 @@ public bool IsSanityChecked(IEnumerable<string> items)
             return true;
         }
 
-        public async Task<bool> CreateFeedContainer(string relativePath)
+        public async Task<bool> CreateFeedContainer()
         {
             ValidateContainerName(ContainerName);
-            string url = $"{FeedUrl}?restype=container";
+            string url = $"{FeedContainerUrl}?restype=container";
             using (HttpClient client = new HttpClient())
             {
-                Tuple<string, string> headerBlobType = new Tuple<string, string>("x-ms-blob-public-access", "blob");
-                List<Tuple<string, string>> additionalHeaders = new List<Tuple<string, string>>() { headerBlobType };
+                List<Tuple<string, string>> additionalHeaders = null;
+                if (!string.IsNullOrWhiteSpace(SecurePath))
+                {
+                    Tuple<string, string> headerBlobType = new Tuple<string, string>("x-ms-blob-public-access", "blob");
+                    additionalHeaders = new List<Tuple<string, string>>() { headerBlobType };
```

Comment:
Why do we need the additional header?

Reply:
I think I understand this now. You only want to add this bit if it is a public feed. I think have the property be IsPublic is a little better then doing the negative.